.github/workflows/master.yml

```diff
@@ -28,27 +28,24 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: '0'
-      - name: Push Docker Image to Github Registry
-        uses: whoan/docker-build-with-cache-action@v5
+      - id: cache-docker
```

Comment:
Any chance we can also push the image to github registry in the master workflow only? They have finally added support for anonymous pull (which was why we had to stick with dockerhub)...

https://docs.github.com/en/packages/getting-started-with-github-container-registry/about-github-container-registry

I'm ok if you want this to go in a separate PR..

Reply:
no, we don't need that because in the end everyone is using dockerhub registry to pull

Reply:
Let's do it in seprate pr